podcasts/Onboarding/Plus/UpgradeExperiment/PlusPaywallContainer.swift

```diff
@@ -0,0 +1,126 @@
+import SwiftUI
+import PocketCastsUtils
+
+struct PlusPaywallContainer: View {
+    @ObservedObject var viewModel: PlusLandingViewModel
+
+    @State var presentSubscriptionView = false
+
+    private let type: ContainerType
+    private let subscriptionInfo: PlusPricingInfoModel.PlusProductPricingInfo?
+    private let tier = UpgradeTier.plus
+
+    private var hasBottomSafeArea: Bool {
+        !UIDevice.current.isiPad() && safeAreaBottomHeight > 0
+    }
+
+    private var safeAreaBottomHeight: CGFloat {
+        (SceneHelper.connectedScene()?.windows.first(where: \.isKeyWindow)?.safeAreaInsets.bottom ?? 0)
```

Comment:
I'm not sure if it fits here but I believe you can access the bottom safe area from `GeometryReader.safeAreaInsets`

Reply:
Thanks @bjtitus ! TBH I used the same method of the current paywall, but I'll give this a try!